timon/run.py

```diff
@@ -183,6 +187,7 @@ async def run_loop(options, cfg, run_once_func=run_once, t00=None):
             break
         if notifiers:
             async with asyncio.TaskGroup() as async_tg:
+                # note: is a list of `Notifier.notify` (see `run_once`)
```

Comment:
perhaps instead of:

`note: is a list`

we could write.


`async_tg is a list`

not sure the word 'note:' adds important information

Reply:
yeah, okay- why is this getting so picky? I literally added comments outside the code I touched and you won't even say thanks :<